src/databricks/labs/ucx/source_code/linters/jobs.py

```diff
@@ -37,17 +38,28 @@
 logger = logging.getLogger(__name__)
 
 
-class WorkflowLinter:
-    def __init__(
+class WorkflowLinter(CrawlerBase):
```

Comment:
@asnare : This "crawler" does not fit the usual crawler format as it "crawls" multiple resources: `JobProblems`, `UsedTable` and `DirectFsAccess`. You looked into this. What is your conclusion on handling this.

Reply:
This was a very difficult problem to solve: I'm not sure I reached a conclusion. I had a sense that we could use some sort of "composite" crawler than produced the different things all at once and then split them up, but never really went through the detailed effort of figuring out the details to graft those two worlds together.